test/CMakeLists.txt

```diff
@@ -19,3 +19,7 @@ add_subdirectory(unit-modcc)
 # Microbenchmarks.
 # Builds: one executable per microbenchmark under ubench/.
 add_subdirectory(ubench)
+
+# 'check' target executes tests.
+add_custom_target(check COMMAND unit)
```

Comment:
1. Comment seems to repeat the code. 
2. Comment seems to miss the important bit: We _need_ this to be called `check` since someone external relies on it, right?